sidecar/nginx/lua/amalgam8.lua

```diff
@@ -476,6 +478,7 @@ function Amalgam8:new()
 
    self.myname = myname
    self.mytags = mytags
+   self.myip = "0.0.0.0"
```

Comment:
Why can't you fetch the IP here?
The IP address of the container will not change once launched. The init function is executed when nginx is (re)started in the container.